en/docs/learn/admin-api.md

```diff
@@ -1,70 +1,73 @@
-Sign in to the API Publisher as a user whose role includes `Internal/publisher`. Follow the steps given below: 
+The page explains how to deploy the administration API and try out the flow.
 
-1. Sign in to the API Publisher portal at `https:// <WSO2_OB_APIM_HOST>:9443/publisher` with `creator/publisher` privileges.
-   You can use the credentials for `mark@gold.com`.
-     ![sign_into](../assets/img/get-started/quick-start-guide/sign-in.png)
+1. Sign in to the API Publisher portal at `https:// <APIM_HOST>:9443/publisher` with `creator/publisher` privileges.
+     
+      ![sign_into](../assets/img/get-started/quick-start-guide/sign-in.png)
 
 2. In the homepage, go to **REST API** and select **Import Open API**. 
+     
      ![let's_get_started](../assets/img/get-started/quick-start-guide/lets-get-started.png)
 
-3. Select **OpenAPI File/Archive**. ![create-an-api](../assets/img/get-started/quick-start-guide/create-an-api.png)
+4. Select **OpenAPI File/Archive**. ![create-an-api](../assets/img/get-started/quick-start-guide/create-an-api.png)
 
```

Comment:
check the numbering of this page. it might not reflect in the site, check this .md file though